### PR TITLE
Add additional features to PHP 8.3 release page

### PR DESCRIPTION
**Pull request by cookieguru**
* Add examples:
  * `php -l` supports multiple files.  This is not currently documented in the manual; see https://github.com/php/doc-en/pull/2917
* Add new features:
  * ini fallback syntax.  Not documented: https://github.com/php/doc-en/issues/2796
  * Anonymous readonly classes, same
* Add BC breaks:
  * SQLite3's error mode
  * Add link to DateTime exceptions RFC

The following features are still not listed on this page:
* Arbitrary static variable initializers
* Changes to Traits and Static Properties

### CODE REVIEW

**Comment by cookieguru** [comment]
I used a machine translation for Russian; if @saundefined or others can proofread that would be appreciated

**Comment by cookieguru** [comment]
Same help requested from @sy-records

File: releases/8.3/release.inc [hunk 1]
```diff
@@ -440,6 +440,40 @@ PHP
...
+No syntax errors detected in bar.php
+						</code>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<div class="php8-compare__content">
+		    <?= message('command_line_linter_description', $lang) ?>
+		</div>
```
Comment (sy-records) [inline releases/8.3/release.inc:475]: Spaces should be used instead of tabs.

btw, I'm not sure if it should be added to the release page or not.

Comment (cookieguru) [reply releases/8.3/release.inc]: Changed to spaces.  Not sure why the [coding standards check](https://github.com/php/web-php/blob/ee1ddcc81aeb3f83576ae7a839d53963be8c8fb2/.github/workflows/integrate.yaml#L13) didn't catch it

File: releases/8.3/languages/ru.php [hunk 2]
```diff
@@ -19,6 +19,8 @@
...
     'randomizer_getfloat_nextfloat_description' => '<p>Из-за ограниченной точности и неявного округления чисел с плавающей точкой генерация несмещённого числа, лежащего в определённом интервале, является нетривиальной задачей, а пользовательские решения могут давать смещённые результаты или числа, выходящие за пределы требуемого диапазона.</p><p>Класс Randomizer был расширен двумя методами, позволяющими генерировать случайные числа с плавающей точкой несмещённым образом. Метод <code>Randomizer::getFloat()</code> использует алгоритм γ-секции, который был опубликован в <a href="https://doi.org/10.1145/3503512" target="_blank" rel="noopener noreferrer">Drawing Random Floating-Point Numbers from an Interval. Frédéric Goualard, ACM Trans. Model. Comput. Simul., 32:3, 2022.</a></p>',
     'dynamic_class_constant_fetch_title' => 'Динамическое получение констант класса',
+    'command_line_linter_title' => 'Линтер командной строки поддерживает несколько файлов',
+    'command_line_linter_description' => '<p>Линтер командной строки теперь принимает переменный ввод имен файлов для проверки.</p>',
```
Comment (saundefined) [inline releases/8.3/languages/ru.php:23]:
```suggestion
    'command_line_linter_description' => '<p>Линтер командной строки теперь принимает нескольно имён файлов для проверки.</p>',
```

**Review by saundefined — APPROVED** [review]
Looks good, thanks!

**Review by sy-records — APPROVED** [review]

**Comment by saundefined** [comment]
@cookieguru can you also add new entries for pt_BR using machine translation? then I'll merge

**Comment by cookieguru** [comment]
@saundefined done

**Comment by saundefined** [comment]
@cookieguru thanks a lot!

**Pull request merged**